internal/extproc/translator/translator.go

```diff
@@ -12,6 +12,19 @@ import (
 	"github.com/envoyproxy/ai-gateway/internal/extproc/router"
 )
 
+var (
+	statusHeaderName       = ":status"
+	contentTypeHeaderName  = "content-type"
+	awsErrorTypeHeaderName = "x-amzn-errortype"
+	jsonContentType        = "application/json"
+	openAIBackendError     = "OpenAIBackendError"
+	awsBedrockBackendError = "AWSBedrockBackendError"
+)
+
+func isGoodStatusCode(code int) bool {
+	return code >= 200 && code < 300
+}
```

Comment:
This needs a detailed comments as the meaning is not obvious without any specific knowledge about how is this used. Like which status codes will be returned by who and when these things happen